schemas/event.yml

```diff
@@ -195,3 +195,32 @@
 
         This is mainly useful if you use more than one system that assigns
         risk scores, and you want to see a normalized value across all systems.
+
+    - name: facility
+      level: core
+      type: long
+      example: 1
+      description: >
+        Value parsed from messages adhering to RFC 5424 or RFC 3164.
+        It represents the process the event has originated from.
+
+    - name: facility_label
+      level: extended
+      type: keyword
+      example: kernel
+      description: >
+        Human readable format of `event.facility`.
+
+    - name: priority
+      level: core
+      type: long
+      example: 1
+      description: >
+        Value parsed from messages adhering to RFC 5424 or RFC 3164.
+
+    - name: priority_label
+      level: extended
+      type: keyword
+      example: Informational
+      description: >
+        Human readable format of `event.priority`.
```

Comment:
Rather than "Human readable", can we add "text-based" or something similar?  Humans can read numbers too :-)

Reply:
The `Informational` example here is an example of a **severity** label (which maps to `log.level` in ECS).

I don't think there is such a thing as a priority label in Syslog. The priority number is the one made up of `8*facility + severity`.

I think we need to remove this field.